src/keys/rsa-class.js

```diff
@@ -3,22 +3,18 @@
 const multihashing = require('multihashing-async')
 const protobuf = require('protons')
 const bs58 = require('bs58')
-const nextTick = require('async/nextTick')
 
 const crypto = require('./rsa')
 const pbm = protobuf(require('./keys.proto'))
-require('node-forge/lib/sha512')
```

Comment:
I believe @hugomrdias added this for reducing the bundle size: https://github.com/libp2p/js-libp2p-crypto/commit/8d8294dc3f9b15d827536f5f79fb512eaf4384a9#diff-17ddd891dd4a459b2c549aebf399e5edR10

Should we really remove this? If not, we should document why we have this

Reply:
Good catch, removing this increased the bundle size by 400kb(!)(!!!!!!!).  I have put it back in.